src/server/ntpcontroller.cpp

```diff
@@ -81,8 +99,3 @@ void NtpController::setConnmanProperty(QString key, QString value)
                   value.toStdString().c_str());
```

Comment:
It does indeed fail to compile. You can get the arguments with `QDBusMessage::arguments()`, and make a string from the `QList<QVariant>` for logging purposes.